### PR TITLE
Updated Error Handling

### PR DESCRIPTION
**Pull request by TheEmperor342**

### CODE REVIEW

**Comment by itsmebirdie** [comment]
acknowledged.

File: Configuration/ModConfig.json [hunk 1]
```diff
@@ -1 +1,5 @@
-{}
\ No newline at end of file
+{
+    "811828700844654592": {
+        "ModEnabled": true
+    }
+}
```
Comment (itsmebirdie) [inline Configuration/ModConfig.json]: Removing this configuration file.

File: main.py [hunk 2]
```diff
@@ -17,7 +17,7 @@
 ## ==> CREATING BOT AND IMPORTING COGS
 #############################################################################################
 
-bot = commands.Bot(command_prefix=">", intents=discord.Intents.all(),case_insensitive=True)
+bot = commands.Bot(command_prefix="!", intents=discord.Intents.all(),case_insensitive=True)
```
Comment (itsmebirdie) [inline main.py:20]: Do you really think it's okay to change the prefix?

**Review by itsmebirdie — COMMENTED** [review]
Code Review completed.
Moving forward to testing.

**Comment by itsmebirdie** [comment]
Approved for merge.
Nice work!

File: main.py (hunk 2 above)
Comment (TheEmperor342) [reply main.py]: I have both assistant and the testing bot in my testing server

That's why I used ! As a prefix for testing purposes

Aaand I forgot to undo it

**Pull request merged**